languages/python/sqlalchemy-oso/tests/test_post_relationship.py

```diff
@@ -620,13 +621,18 @@ def test_empty_constraints_in(session, oso, tag_nested_many_many_test_fixture):
         # SQLAlchemy 1.4.
         true_clause = " AND 1 = 1"
 
+    if USING_SQLAlchemy_v2_0:
+        tables = "tags, post_tags"
+    else:
+        tables = "post_tags, tags"
```

Comment:
It matters because we do a string comparison of the SQL we get back. SQLAlchemy 2.0 reordered these a little. We have this pattern elsewhere. 

Would be nice if we had snapshots like with insta!